File: ambeth/Ambeth.Service/ambeth/model/IServiceResult.cs
using System;
using System.Collections.Generic;
using De.Osthus.Ambeth.Merge.Model;
using De.Osthus.Ambeth.Annotation;

namespace De.Osthus.Ambeth.Cache.Model
{
    [XmlType(Name = "IServiceResult", Namespace = "http://schemas.osthus.de/Ambeth")]
    public interface IServiceResult
    {
        IList<IObjRef> ObjRefs { get; }

        Object AdditionalInformation { get; }
    }
}

File: jambeth/jAmbeth.Service/src/de/osthus/ambeth/cache/model/IServiceResult.java
package de.osthus.ambeth.cache.model;

import java.util.List;

import de.osthus.ambeth.annotation.XmlType;
import de.osthus.ambeth.merge.model.IObjRef;

@XmlType
public interface IServiceResult
{
	List<IObjRef> getObjRefs();

	Object getAdditionalInformation();
}
